pkg/apply/event/type_string.go

```diff
@@ -15,11 +15,12 @@ func _() {
 	_ = x[StatusType-4]
 	_ = x[PruneType-5]
 	_ = x[DeleteType-6]
+	_ = x[WaitType-7]
 }
 
-const _Type_name = "InitTypeErrorTypeActionGroupTypeApplyTypeStatusTypePruneTypeDeleteType"
+const _Type_name = "InitTypeErrorTypeActionGroupTypeApplyTypeStatusTypePruneTypeDeleteTypeWaitType"
 
```

Comment:
You need to rebase this PR. The change to this file has been in the master already: https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/apply/event/type_string.go#L21

Reply:
done